mrmustard/lab/states.py

```diff
@@ -106,16 +104,20 @@ def means(self):
 class SqueezedVacuum(Parametrized, State):
     r"""
     The N-mode squeezed vacuum state. Equivalent to applying a squeezing gate to the vacuum state:
-    >>> SqueezedVacuum(r=0.5, phi=0.2) == Vacuum(1) >> Sgate(r=0.5, phi=0.2)
-    True
+
+    .. code::
+      >>> SqueezedVacuum(r=0.5, phi=0.2) == Vacuum(1) >> Sgate(r=0.5, phi=0.2)
+      True
 
     Parallelizable over r and phi:
-    >>> SqueezedVacuum(r=[1.0, 2.0], phi=[-1.0, -2.0]) == SqueezedVacuum(r=1.0, phi=-1.0) & SqueezedVacuum(r=2.0, phi=-2.0)
-    True
+    .. code::
+      >>> SqueezedVacuum(r=[1.0, 2.0], phi=[-1.0, -2.0]) == SqueezedVacuum(r=1.0, phi=-1.0) & SqueezedVacuum(r=2.0, phi=-2.0)
+      True
 
     Can be used to model a heterodyne detection with result 0.0:
-    >>> Gaussian(2) << SqueezedVacuum(r=10.0, phi=0.0)[1]  # e.g. homodyne on x quadrature on mode 1 with result 0.0
-    # leftover state on mode 0
+    .. code::
+      >>> Gaussian(2) << SqueezedVacuum(r=10.0, phi=0.0)[1]  # e.g. homodyne on x quadrature on mode 1 with result 0.0
+      # leftover state on mode 0
 
     Args:
         r (float): The squeezing magnitude.
```

Comment:
Same for other args.
```suggestion
    Args:
        r (float): the squeezing magnitude
```